Proofs/SHA512/SHA512Sym.lean

```diff
@@ -47,6 +47,7 @@ abbrev ktbl_addr : BitVec 64 := 0x1b4300#64
 -- set_option profiler true in
 -- set_option profiler.threshold 1 in
 -- set_option pp.deepTerms false in
+set_option trace.Tactic.cse.summary true in
 theorem sha512_block_armv8_1block (s0 sf : ArmState)
   -- (FIXME) Ignore the `stp` instruction for now.
   (h_s0_pc : read_pc s0 = 0x1264c4#64)
```

Comment:
I'm not a big fan of having trace options committed, since they make tend to make the build log quite noisy

Reply:
Discussed alternatively.